apps/web/composables/useContent/index.ts

```diff
@@ -1 +1 @@
-export * from './useContent';
+// export * from './useContent';
```

Comment:
empty files are not allowed. So I guess we have to remove the index file or just comment the sdk section in the composable

Reply:
removed files